templates/php.yml

```diff
@@ -1,26 +1,80 @@
 # PHP
-# Test and package your PHP project.
-# Add steps that run tests, save build artifacts, deploy, and more:
+# Build, package and deploy your PHP project.
+# Add steps that run tests and more:
 # https://docs.microsoft.com/azure/devops/pipelines/languages/php
 
 trigger:
 - master
 
-pool:
-  vmImage: 'ubuntu-latest'
-
 variables:
-  phpVersion: 7.2
-
-steps:
-- script: |
-    sudo update-alternatives --set php /usr/bin/php$(phpVersion)
-    sudo update-alternatives --set phar /usr/bin/phar$(phpVersion)
-    sudo update-alternatives --set phpdbg /usr/bin/phpdbg$(phpVersion)
-    sudo update-alternatives --set php-cgi /usr/bin/php-cgi$(phpVersion)
-    sudo update-alternatives --set phar.phar /usr/bin/phar.phar$(phpVersion)
-    php -version
-  displayName: 'Use PHP version $(phpVersion)'
-
-- script: composer install --no-interaction --prefer-dist
-  displayName: 'composer install'
+  # Azure Resource Manager connection created during pipeline creation
+  azureSubscription: '{{ azureRmConnection.Id }}'
+  
+  # Web app name
+  webAppName: '{{ webAppName }}'
+
+  # Agent VM image name
+  vmImageName: 'ubuntu-latest'
+
+  # Environment name
+  environmentName: "$(Build.DefinitionName)-Prod"
```

Comment:
Does all characters supported by a Build definition Name supported as valid characters for Environment name? Can you check that?

Reply:
> Does all characters supported by a Build definition Name supported as valid characters for Environment name? Can you check that?
I'm not sure if this will be the final format. Usha will check and confirm. It might as well be $(reponame)-Dev.

Reply:
I'm not sure if this will be the final format. Usha will check and confirm. It might as well be $(reponame)-Dev.

Reply:
Ok. You will still need to ensure all reponame values are valid environment name characters.

Reply:
Both Build Definition and repo name allow ".". So, we are using WebApp name as default environment name instead.

Reply:
Webapp name need not be unique (like within ASE and outside ASE). You will probably need to handle that case.